backend/src/services/integration-auth/integration-sync-secret.ts

```diff
@@ -457,6 +457,8 @@ const syncSecretsAWSParameterStore = async ({
   });
   ssm.config.update(config);
 
+  const metadata = z.record(z.any()).parse(integration.metadata);
```

Comment:
Yeah this was what we decided to do for `metadata` for other integrations as well during the migration; it can be improved but since it is this way elsewhere it should be fine.